packages/sdk/src/content_topic.ts

```diff
@@ -0,0 +1,110 @@
+import type { Multiaddr } from "@multiformats/multiaddr";
+import {
+  createDecoder,
+  DecodedMessage,
+  waitForRemotePeer,
+  WakuNode
+} from "@waku/core";
+import {
+  Callback,
+  IDecoder,
+  IFilterSubscription,
+  IMetadata,
+  Protocols
+} from "@waku/interfaces";
+import {
+  contentTopicToPubsubTopic,
+  shardInfoToPubsubTopics
+} from "@waku/utils";
+
+import { createLightNode } from ".";
+
+// Given a Waku node, peer Multiaddr, and content topic, creates a decoder and
+// subscription for that content topic.
+export async function prepareSubscription(
```

Comment:
this function seems to be used within another function -- do we need to export this?

Reply:
fixed in last commit